Dockerfile-v4-21.catalog

```diff
@@ -0,0 +1,22 @@
+# The base image is expected to contain /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
+# TODO migrate to this once available FROM registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.21
+FROM brew.registry.redhat.io/rh-osbs/openshift-ose-operator-registry-rhel9:v4.21
```

Comment:
it's also available at `registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.21`